webpack/scenes/ContentViews/Details/Filters/MatchContentModal/CVRpmMatchContentModal.js

```diff
@@ -41,6 +44,16 @@ const CVRpmMatchContentModal = ({ filterId, onClose, filterRuleId }) => {
       isOpen
       onClose={onClose}
       appendTo={document.body}
+      help={
+        <Popover
+          headerContent={__('Help')}
+          bodyContent={__("Matching RPMs based on your created filter rule. Remember, package filters don't apply to modular packages.")}
```

Comment:
```suggestion
          bodyContent={__("Matching RPMs based on your created filter rule. Remember, RPM filters don't apply to modular packages.")}
```

Same goes for the other mentions of "package".